iOSNativeTemplate/iOSNativeTemplate.xcodeproj/project.pbxproj

```diff
@@ -380,6 +382,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.salesforce.iosnativetemplate.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
iOSNativeTemplate could not be generated anymore because of swift version was not specified.

Reply:
Thank you!  I was going to send a pr for this in the morning.